docs/scaffold/index.md

```diff
@@ -2,5 +2,5 @@
 order: false
 parent:
   order: 2
-  title: Scaffold a Blockchain
+  title: Scaffolding
```

Comment:
oh! I see you favor the gerund active verb -ing. The "Scaffold a Blockchain" is using the same  title construction as "Run a Blockchain"

can we go back to Scaffold a Blockchain? @fadeev

Reply:
right right :)